blockchain/state/doc.go

```diff
@@ -18,5 +18,26 @@
 // This file is derived from core/state/statdb.go (2018/06/04).
 // Modified and improved for the klaytn development.
 
-// Package state provides a caching layer atop the Klaytn state trie.
+/*
+Package state provides a caching layer atop the Klaytn state trie.
+This package is used to read/write stateObject from/to StateDB and it also acts as a bridge between
+the objects and the persistent layer.
+
+Overview of state package
+
+stateObject represents a Klaytn account which is being modified. It is identified by its address.
+Once it is loaded from the persistent layer, it is cached and managed by StateDB.
+StateDB manages stateObjects and operations to them. Most of the operations are done by StateDB,
+instead of direct access to stateObject.
+
+Source Files
```

Comment:
`Source Files` were not processed heading.
![image](https://user-images.githubusercontent.com/13497596/62034689-6d613080-b229-11e9-9d05-e8f3f3ed1ee6.png)

Reply:
Thanks! Fixed!